runlicensecheck.bat

```diff
@@ -0,0 +1,12 @@
+@echo off
+rem runlicensecheck [<output file>]
+rem Runs a license check for python dependencies
+set hereOrig=%~dp0
+set here=%hereOrig%
+if #%hereOrig:~-1%# == #\# set here=%hereOrig:~0,-1%
```

Comment:
Is this perhaps clearer in intent?
```suggestion
if #%here:~-1%# == #\# set here=%here:~0,-1%
```

Reply:
I'm not sure, this is magic sauce at the top of each of our `.bat` files. I'm afraid to change it without updating them all, and concerned this suggestion is different in handling